src/Program.cs

```diff
@@ -1,5 +1,42 @@
-﻿// See https://aka.ms/new-console-template for more information
+﻿using System;
+using Arcane.Framework.Contracts;
+using Arcane.Framework.Providers.Hosting;
+using Arcane.Stream.Cdm;
+using Arcane.Stream.Cdm.Models;
+using Microsoft.Extensions.Hosting;
+using Serilog;
+using Snd.Sdk.Logs.Providers;
+using Snd.Sdk.Metrics.Configurations;
+using Snd.Sdk.Metrics.Providers;
+using Snd.Sdk.Storage.Providers;
+using Snd.Sdk.Storage.Providers.Configurations;
 
-using System;
+Log.Logger = DefaultLoggingProvider.CreateBootstrapLogger(nameof(Arcane));
 
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+int exitCode;
+try
+{
+    exitCode = await Host.CreateDefaultBuilder(args)
+        .AddDatadogLogging((_, _, configuration) => configuration.WriteTo.Console())
```

Comment:
Also, one thing I forgot to mention: could you please add custom logging properties, like [here](https://github.com/SneaksAndData/arcane-stream-rest-api/blob/4fced853c68f544f209641fedf6777f479f67553/src/Program.cs#L23)?